web/app/components/favicon.hbs

```diff
@@ -0,0 +1,17 @@
+{{#if this.fallbackIconIsShown}}
+  <FlightIcon
+    ...attributes
+    data-test-fallback-favicon
+    @name="globe"
```

Comment:
Should we keep the logic in `get-link-icon.ts` (that's currently deleted in this PR) to get a more specific icon?

Reply:
It seemed more confusing than helpful in the fallback scenario, where the job is to communicate "something failed to load." If we're starting with three favicon'd links, e.g.,

🟧 Asana discussion
🌈 Figma prototype
🟦 Algolia docs

and then one day it looks like

🌐 Asana discussion
🌈 Figma prototype
🌐 Algolia docs

Reply:
Got it, makes sense 👍